tests/ast.test.ts

```diff
@@ -41,4 +56,119 @@ describe('ast', () => {
         expect(node).toBeTruthy();
         check(node!);
     });
+
+    test('unchanged file should have equal caches', async () => {
+        const file = mo.file('AST.mo');
+        file.write(actorSource);
+        const [prog0, cache0] = file.parseMotokoTypedWithScopeCache(new Map<string, Scope>());
+        expect(prog0.ast).toBeTruthy();
+        expect(prog0.immediateImports).toEqual(
+            ['.node-motoko/base/moc-0.14.1/Debug.mo'],
+        );
+        expect(Array.from(cache0.keys())).toEqual(
+            ['.node-motoko/base/moc-0.14.1/Debug.mo', '@prim'],
```

Comment:
hardcoded versions?

Reply:
fixed

Reply:
@rvanasa is there a way to dynamically figure this out?

Reply:
Okay, looks like taken care of in b42f767462b48fb0602fe7d26f80571a8df5b325.